apps/src/music/views/MusicView.jsx

```diff
@@ -884,6 +887,21 @@ class UnconnectedMusicView extends React.Component {
     });
     if (this.props.isFirstAttempt) {
       this.props.sendAttemptReport();
```

Comment:
Some food for thought for SL Lab folks (@molly-moen @breville)...there seem to be a few common actions that happen on first run (for example here, attempt report and now the level activity metric). We've been careful to not make assumptions about how labs "run" their project since that differs from level to level, but may be worth thinking about at least pulling these common actions into a common action path (util function, redux thunk) that labs can then just call once in their "run" action or equivalent.